#! /bin/sh /usr/share/dpatch/dpatch-run
## autoconf-upgrade.dpatch by  <rodrigo@nul-unu.com>
##
## DP: Properly check for headers and avoid autoconf >=2.50 warnings

@DPATCH@
diff -urNad sawfish-1.3+svn4194~/configure.in sawfish-1.3+svn4194/configure.in
--- sawfish-1.3+svn4194~/configure.in	2007-05-02 18:59:11.000000000 -0500
+++ sawfish-1.3+svn4194/configure.in	2007-05-02 19:03:52.048606925 -0500
@@ -90,7 +90,8 @@
 _cppflags="${CPPFLAGS}"
 CPPFLAGS="${CPPFLAGS} ${X_CFLAGS}"
 
-AC_CHECK_HEADERS(X11/SM/SMlib.h X11/extensions/Xdbe.h)
+AC_CHECK_HEADERS(X11/SM/SMlib.h X11/extensions/Xdbe.h, [], [],
+        [#include <X11/Xlib.h> ])
 
 dnl Checks for X extensions/libraries
 XINERAMA_LIBS=""
@@ -110,7 +111,8 @@
 
 AC_CHECK_LIB(Xrandr, XRRUpdateConfiguration,
 	     [XRANDR_LIBS="-lXrandr"
-	      AC_CHECK_HEADERS(X11/extensions/Xrandr.h)],
+	      AC_CHECK_HEADERS(X11/extensions/Xrandr.h, [], [],
+                [#include <X11/Xlib.h> ])],
 	     [],[$X_LIBS -lX11 -lXext])
 
 dnl Try using pkg-config first, since it looks like Xft2 may fail our
